src/tests/ftest/launch.py

```diff
@@ -63,7 +63,6 @@ def __del__(self):
 TEST_LOG_FILE_YAML = "./data/daos_avocado_test.yaml"
 BASE_LOG_FILE_YAML = "./data/daos_server_baseline.yaml"
 SERVER_KEYS = (
-    "test_machines",
```

Comment:
In addition the `daos_servers` entry can be removed - it used to be used by soak.yaml

Reply:
Done